tooling/utils.js

```diff
@@ -26,3 +26,9 @@ export const isValidFoundryImage = str => {
 		str.split(".").map(it => it.trim()).filter(Boolean).at(-1)
 	);
 };
+
+// Based on Foundry CLI's `getSafeFilename`
+// See: https://github.com/foundryvtt/foundryvtt-cli/blob/c9d14019147997dc7f45382dc8f6842f66aa3f5d/lib/package.mjs#L546
+export const getSafeFilename = (filename) => {
```

Comment:
Previously we used an off-the-shelf filename sanitizer. As Foundry has rolled their own, we follow suit.

Reply:
Dammit, why can't they export it

Reply:
Agreed, it is a shame. Same for the rest of that CLI file, there are lots of useful little functions